cnf-certification-test/accesscontrol/suite.go

```diff
@@ -611,3 +618,37 @@ func Test1337UIDs(env *provider.TestEnvironment) {
 
 	testhelper.AddTestResultLog("Non-compliant", badPods, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+func testContainerSCC(env *provider.TestEnvironment) {
+	var containerSCC securitycontextcontainer.ContainerSCC
+	const istioProxyContainerUID = 1337
+	var badCut []string
+	for _, pod := range env.Pods {
+		containerSCC.HostIPC = pod.Spec.HostIPC
```

Comment:
The container level parameters should take precedence over the pod level ones, only if they are defined. looks like the GetContainerSCC is always overwriting the values collected at the pod level. To track whether a parameter is set of not, maybe we could use pointers instead of values. So that it it is null, the param is not se, for example:
```
	HostDirVolumePlugin    *bool
	HostIPC                *bool
	HostNetwork            *bool
	HostPID                *bool
	HostPorts              *bool
	PrivilegeEscalation    *bool
	PrivilegedContainer    *bool
	RunAsUser              *bool
	ReadOnlyRootFilesystem *bool
	RunAsNonRoot           *bool 
	fsGroup               *bool 
	seLinuxContext:       *bool 
	volumes               string[]
```

Other values missing from pod level:
- runAsGroup
- runAsNonRoot
- selinuxOptions